flutter/lib/src/default_integrations.dart

```diff
@@ -444,6 +410,30 @@ class _LoadAndroidImageListIntegrationEventProcessor extends EventProcessor {
 
     return event;
   }
+
+  static bool needsSymbolication(SentryEvent event) {
+    final frames = event.exceptions?.first.stackTrace?.frames;
+    if (frames == null) return false;
+    return Platform.isAndroid
```

Comment:
You can use `options.platformChecker.platform.isAndroid` instead so it's possible to unit test it.